src/lang/en-US.ts

```diff
@@ -18,6 +18,20 @@ const enUSMessages: ResolvedIntlConfig['messages'] = {
     'cta.download': `Download`,
     'cta.login': `Login`,
     'cta.clickHere': `click here`,
+    'cta.moreDetails': `More Details`,
+
+    // Terms
+    'terms.connectors': `Connectors`,
+
+    // Data
+    'data.name': `Name`,
+    'data.description': `Description`,
+    'data.type': `Type`,
+    'data.maintainer': `Maintainer`,
+    'data.updated_at': `Last Updated`,
```

Comment:
The idea here is that we'll be displaying keys of data from the server in lots of places. So we should have standard keys (when possible) to match them. The second portion of the key is the key from the server. Not truly sure if we want this kind of dependency though.